### PR TITLE
Adds docs for v2 UI builds

### PR DESCRIPTION
**Pull request by stevsmit**

### CODE REVIEW

File: modules/proc_use-quay-build-dockerfiles.adoc [hunk 1]
```diff
@@ -204,43 +207,31 @@ Use the following procedure to create a Build trigger.
...
-. When prompted, select a repository. Then, click *Continue*. 
-
-. Configure the trigger by selecting one of the following options:
-
-.. *Trigger for all branches and tags (default1)*. By selecting this option, a container image for each commit across all branches and tags is created. 
-
-.. *Trigger only on branches and tags matching a regular expression*. By selecting this option, only container images for a subset of branches and/or tags are built. 
-
-. Click *Continue*. 
+. Enter a repository name, for example, `\https://github.com/<username>/testrepo`. Then, click *Next*.
```
Comment (huweihua-redhat) [inline modules/proc_use-quay-build-dockerfiles.adoc:210]: If we take `github` as example,  the enter page should be like this 
Screenshot 2024-03-14 at 15 51 36
The github credentials have been configured in config.yaml ahead.  So customer doesn't need to input repository name. 
If we take `custom Git repository` as example,  now customer needs to input repository name,  but the format is `git@github.com:<username>/<repo>.git`.
Screenshot 2024-03-14 at 15 52 39

File: modules/proc_use-quay-build-dockerfiles.adoc [hunk 2]
```diff
@@ -204,43 +207,31 @@ Use the following procedure to create a Build trigger.
...
-. Check for any verification warnings. If necessary, fix the issues before clicking *Continue*. 
+. Optional. Choose an optional robot account. This allows you to pull a private base image during the build process. If you know that a private base image is not used, you can skip this step. 
 
-. When prompted with *Ready to go!*, click *Continue*. You are redirected to a confirmation page.
+. Click *Next*. Check for any verification warnings. If necessary, fix the issues before clicking *Finish*. 
 
-. Save the SSH Public Key, then click *Return to <organization_name>/<repository_name>*. You are redirected to the *Builds* page of your repository. 
+. You are alerted that the trigger has been successfully activated. Note that using this trigger requires the following actions:
+* You must give the following public key read access to the git repository.
+* You must set your repository to `POST` to the following URL to trigger a build.
```
Comment (huweihua-redhat) [inline modules/proc_use-quay-build-dockerfiles.adoc:232]: Depending on the line 231 and 232,  I guess you are taking `custom Git repository` as example.  
For `github repository push` trigger, customer doesn't need to do these,  quay has done these steps in the back-end with the credentials in config.yaml.

File: modules/proc_use-quay-build-dockerfiles.adoc [hunk 3]
```diff
@@ -167,22 +167,25 @@ endif::[]
 
 .Procedure
 
-. On the *Builds* page, click *Start New Build*. Alternatively, you can click the `+` icon to reveal a drop-down menu. 
+. On the *Builds* page, click *Start New Build*.
 
-. When prompted, click *Select File* to upload a Dockerfile or an archive (for example, a `.tar.gz` or `.zip` file) that contains a Dockerfile at the root directory.
+. When prompted, click *Upload Dockerfile* to upload a Dockerfile or an archive (for example, a `.tar.gz` or `.zip` file) that contains a Dockerfile at the root directory.
```
Comment (bcaton85) [inline modules/proc_use-quay-build-dockerfiles.adoc:172]: > an archive (for example, a `.tar.gz` or `.zip` file)
The new UI currently doesn't support this. We're debating how much this is actually used.

File: modules/proc_use-quay-build-dockerfiles.adoc (hunk 1 above)
Comment (stevsmit) [reply modules/proc_use-quay-build-dockerfiles.adoc:210]: @huweihua-redhat added the following information: 

[NOTE]
====
The following procedure assumes that you have not included Github credentials in your `config.yaml` file.
====

. Enter a custom Git repository name, for example, `\git@github.com:<username>/<repo>.git`. Then, click *Next*.

File: modules/proc_use-quay-build-dockerfiles.adoc (hunk 2 above)
Comment (stevsmit) [reply modules/proc_use-quay-build-dockerfiles.adoc:232]: I am going to use a custom Git repo for this example :) I have changed some wording to reflect that.

**Comment by huweihua-redhat** [comment]
LGTM, thanks @stevsmit

**Pull request merged**